libraries/Microsoft.Bot.Builder.Dialogs.Debugging/CodeModels/CodePoint.cs

```diff
@@ -50,7 +50,7 @@ object ICodePoint.Evaluate(string expression)
             var result = exp.TryEvaluate(DialogContext.State);
             if (!string.IsNullOrEmpty(result.error))
             {
-                throw new Exception(result.error);
+                throw new ArgumentException(result.error);
```

Comment:
Shouldn't this one be `InvalidOperationException`?  This is not an error with the argument itself.

Reply:
The exception is the result of an evaluation of the argument `expression`, that's why I used `ArgumentException`.

Reply:
But, in the rest of the PRs, you are using _InvalidOperationException_ for errors in expression evaluations.

Reply:
Yes, there was a consistency issue. Thanks for pointing it out!